### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by bugraderre**
Split responsibility of documentation in GitHub and Proofdock docs. 
* ``Contrib`` modules declare the usage chapter
* ``Contrib`` modules reference the usage chapter from main README
* The control panel is described in the Proofdock docs and therefore the user is forwarded to those docs.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,29 +16,29 @@ the rules of the DCO before submitting a PR.
...
 ### Develop
 
-If you wish to develop on this project, make sure to install the development
-dependencies. But first, [create a virtual environment][venv] and then install
-those dependencies.
+Developing on this project requires you to set up a development environment:
 
-[venv]: http://chaostoolkit.org/reference/usage/install/#create-a-virtual-environment
+1. We strongly encourage you to [set up a virtual environment][venv].
+2. Install the dev dependencies in  environment. The dev dependencies instruct pip to install the testing package `pytest`, the code style checker `flake8` and other development-related packages:
```
Comment (mkaszub) [inline CONTRIBUTING.md:22]: the environment?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,29 +16,29 @@ the rules of the DCO before submitting a PR.
...
+   ```console
+   $ pip install -r requirements-dev.txt
+   ```
+3. Deploy the project source in [Development Mode][setuptools_development_mode]. This deployment is done in such a way that changes to the project source are immediately available in your development environment, without needing to run a build or install step after each change:
```
Comment (mkaszub) [inline CONTRIBUTING.md:26]: without need? instead of without needing?

File: README.md [hunk 3]
```diff
@@ -3,52 +3,80 @@
 ![CI](https://github.com/proofdock/chaos-middleware-python/workflows/CI/badge.svg?branch=master)
 [![Python versions](https://img.shields.io/pypi/pyversions/proofdock-chaos-middleware-python.svg)](https://www.python.org/)
 
-Practice chaos engineering on Python applications. This project is a collection of features to inject application level failures. Various Python frameworks are supported.
+Practice chaos engineering on Python applications. This project is a collection of features to inject failures at the application level. Various Python frameworks are supported.
```
Comment (mkaszub) [inline README.md:6]: hmm maybe "with" Python applications? "on" I guess is as wel okeyish :D

Comment (bugraderre) [reply README.md:6]: Fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (bugraderre) [reply CONTRIBUTING.md:26]: I took the text from the setuptools description. I assume their English is better than mine :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (bugraderre) [reply CONTRIBUTING.md:22]: Good catch

**Pull request merged**